pkg/cloud/aws/services/certificates/certificates.go

```diff
@@ -69,73 +68,78 @@ type Config struct {
 
 // ReconcileCertificates generate certificates if none exists.
 func (s *Service) ReconcileCertificates() error {
-	klog.V(2).Infof("Reconciling certificates for cluster %q", s.scope.Cluster.Name)
-	clusterCAKeyPair, err := getOrGenerateCACert(&s.scope.ClusterConfig.CAKeyPair, clusterCA)
-	if err != nil {
-		return errors.Wrapf(err, "Failed to generate certs for %q", clusterCA)
-	}
-	s.scope.ClusterConfig.CAKeyPair = clusterCAKeyPair
+	s.scope.V(2).Info("Reconciling certificates", "cluster-name", s.scope.Cluster.Name, "cluster-namespace", s.scope.Cluster.Namespace)
```

Comment:
Should this be `s.scope.Logger` for consistency?

Reply:
Ooooops I think `s.scope.V(2)` is a little cleaner and what I meant to do, do you prefer `s.scope.Logger` or `s.scope.V(n)`?

Reply:
I don't mind either way 😄, I think we should pick one and suggest it as default just for consistency

Reply:
totally, thanks for pointing out inconsistency! This was a mind numbing task haha